api/MediaStream.json

```diff
@@ -27,10 +27,10 @@
             "version_added": "12"
           },
           "firefox": {
-            "version_added": "15"
```

Comment:
Have you checked where this data came from, and if it's consistent with APIs that take or return MediaStreams?

Maybe this interface has been prefixed, or the interface object hasn't always been exposed?

Reply:
Per https://developer.mozilla.org/en-US/docs/Web/API/Navigator/getUserMedia#browser_compatibility it has to be available since 17 at least.

Also it should probably match MediaStreamTrack.